.goreleaser.yml

```diff
@@ -1,20 +1,28 @@
 env:
   - GO111MODULE=on
+  - KUBEBENCH_CFG=/etc/kube-bench/cfg
 builds:
   - main: main.go
     binary: kube-bench
     goos:
       - linux
     goarch:
       - amd64
+    ldflags:
+     - "-X github.com/aquasecurity/kube-bench/cmd.KubeBenchVersion={{.Version}}"
+     - "-X github.com/aquasecurity/kube-bench/cmd.cfgDir={{.Env.KUBEBENCH_CFG}}"
 # Archive customization
 archive:
   format: tar.gz
+  files:
+    - "cfg/**/*"
 nfpm:
   vendor: Aqua Security
   description: "The Kubernetes Bench for Security is a Go application that checks whether Kubernetes is deployed according to security best practices"
   license: Apache-2.0
   homepage: https://github.com/aquasecurity/kube-bench
+  files:
+    "cfg/**/*": "/etc/kube-bench/cfg"
```

Comment:
I wonder if `cfg` is still required if files are already in `/etc/kube-bench` ?

You may want to align the location of the configuration between container and rpm (and other). I can see in the README that the container expects the files in `/opt/kube-bench/cfg/`.

IMHO: `/etc/kube-bench` sounds a better place to me.